reference/5.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md

```diff
@@ -235,6 +235,41 @@ The environment variables that store preferences include:
 
     For more information, see [about_Execution_Policies](about_Execution_Policies.md).
 
+- PSModuleAnalysisCachePath
+
+  PowerShell provides control over the file that is used to cache data about a
+  module, such as the commands it exports.
+
+  By default on Windows, this cache is stored in the file
+  `${env:LOCALAPPDATA}\Microsoft\Windows\PowerShell\ModuleAnalysisCache`. The cache is typically read
```

Comment:
Strictly speaking, this is not entirely correct. For example, on the Azure App Service sandbox, the `$env:LOCALAPPDATA` points to a certain existing location (`D:\local\LocalAppData`), but this is not where PowerShell creates the cache files. Instead, it takes the location from `Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData)` (returns an empty string in on the sandbox environment).

Reply:
So, in PowerShell terms, the location is: 
``` PowerShell
`$([System.Environment]::GetFolderPath([System.Environment+SpecialFolder]::LocalApplicationData))\Microsoft\Windows\PowerShell\ModuleAnalysisCache`
```
I realize this looks a bit verbose for documentation, but this is the truth.

By the way, if the code is changed to _actually_ use `$env:LOCALAPPDATA`, the https://github.com/Azure/azure-functions-powershell-worker/issues/365 issue will also be solved.

Reply:
I discussed this with Sean and updated it.  The technically correct path is too long and confusing to users and isn't really applicable to most users as the env var is correct except in the sandbox.

Reply:
Agreed. The word "normally" is very appropriate here.